recipes/libtiff/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "4.0.6" %}
+
+package:
+    name: libtiff
+    version: {{ version }}
+
+source:
+    fn: tiff-{{ version }}.tar.gz
+    url: ftp://ftp.remotesensing.org/pub/libtiff/tiff-{{ version }}.tar.gz
+    md5: d1d2e940dea0b5ad435f21f03d96dd72
+    patches:
+        - def_snprintf.patch  # [win]
```

Comment:
It seems that I wrote it: https://github.com/conda/conda-recipes/blob/0dabe789218ef58df023154dfc6482791b336451/libtiff/def_snprintf.patch .  I vaguely recall something about it.  I guess if you're worried about it, leave it out and see what happens.  I think you run into either duplicate definition of snprintf or lack of it, depending on the compiler you're using.